doc/release/0.2.rst

```diff
@@ -20,6 +20,13 @@ ARMI v0.2.2
 ===========
 Release Date: 2022-01-19
 
+What's new in ARMI v0.2.2
+-------------------------
+#. Improved type hinting
+#. Flushed out the ability to build the docs as PDF
+#. Material modifications can now be made per-component
+#. The ``loadOperator`` method now has the optional ``allowMissing`` argument
```

Comment:
Make a note that this allows for handling backwards compatibility with database loading if parameters are removed from the data model.